pnpm-workspaces/packages/one/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
+
+ts_project(
+    name = "one_ts",
+    srcs = ["src/main.ts"],
+    tsconfig = "//:tsconfig",
+    deps = [
+        "//:node_modules/@types/node",
+    ],
+)
+
+npm_package(
+    name = "one",
+    srcs = [
+        "package.json",
+        ":one_ts",
+        "//:tsconfig",
```

Comment:
I don't think this has to be in the `npm_package`? It should only be required for compiling the ts?